CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# v1.9.4
+
+### `@liveblocks/react-comments`
+
+- Implement polling on `useThreads` hook.
```

Comment:
Since we had it before, I think we should say something closer to "Add back polling to `useThreads` hook" or "Fix polling on `useThreads` hook"

Also, I can't comment on it but there's a typo on 1.9.3's update below: "being be used"

Reply:
Good point. Updated it to `Fix polling on useThreads hook.`

Reply:
Could you change it to "Fix polling on `useThreads` hook."? 🙏 And also fix the typo in the 1.9.3 update, I'd prefer not to open a PR just for that.

Reply:
Done!